_episodes/012-pietro-pascal-santiago.md

```diff
@@ -0,0 +1,86 @@
+---
+layout: post
+title: "RustFest Interviews Triple Feature: Rust Release Engineering; Developing the Developer Tools; Rust in Latin America"
+date: 2015-05-15T16:00:00Z
```

Comment:
This seems wrong

Reply:
Yeah I'm instituting a policy of not bothering to figure out the date until just before I'm about to publish. :P